internal/provider/provider.go

```diff
@@ -373,6 +373,34 @@ func New() *schema.Provider {
 						Optional:      true,
 						ConflictsWith: []string{"command"},
 					},
+					"healthcheck_enabled": {
+						Type:          schema.TypeBool,
+						Description:   "Run a HTTP request periodically to determine the application readiness.",
+						ForceNew:      true,
+						Optional:      true,
+						ConflictsWith: []string{"command"},
+					},
```

Comment:
Can `healthcheck_enabled` be implied by the existence of `healthcheck_url` instead of having it's own value?

Reply:
I'm down for that